src/js/components/Skeleton/stories/GridContainer.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+
+import { Box, Button, Card, Grid, Image, Text} from 'grommet';
+import { FormAdd } from 'grommet-icons';
+
+const labels = [];
+for (let i = 0; i < 10; i += 1) labels.push(`Item ${i}`);
+
+const skeleton = { message: { start: "Loading", end: 'Content Loaded' } };
+
+const Item = ({ title, ...rest }) => (
+  <Card
+    width="531px"
```

Comment:
can we avoid custom px values in the stories and use tshirt size instead?

Reply:
tshirt size here didn't work very well for the example. I've run into that alot with cards in the wild